.gitignore

```diff
@@ -8,6 +8,8 @@ __pycache__/
 # Distribution / packaging
 .Python
 env/
+wg-venv/
+venv-django/
```

Comment:
Create a `~/.gitignore` and put these there. They should not be in the project `.gitignore`.See [this post](https://hackernoon.com/exclude-files-from-git-without-committing-changes-to-gitignore-986fa712e78d) for some justification.

Remember to add this line to your `~/.gitconfig`, `excludesfile = /Users/[your username]/.gitignore` in the `core` section`